api/src/functions/express/app.ts

```diff
@@ -118,9 +119,10 @@ const apiFormationClient = ApiFormationClient(
 );
 
 app.use(bodyParser.json({ strict: false }));
+app.use("/api", userRouterFactory(userDataClient, updateLicenseStatus));
 app.use(
-  "/api",
-  userRouterFactory(userDataClient, updateLicenseStatus, addGovDeliveryNewsletter, addToAirtableUserTesting)
+  "/api/ext",
```

Comment:
typo? incomplete word here. also, i'd prefer to follow the pattern of any sub-routes past /api being defined in the router instead of top-level app

Reply:
actually, never mind. given the change to the index.ts lambda, this makes more sense here because it is required for the auth bypass.

Reply:
i would still rather have a real word than "ext" though